Formula/llnode.rb

```diff
@@ -18,7 +18,6 @@ class Llnode < Formula
   end
 
   depends_on "node" => :build
-  depends_on "python@3.9" => :build
```

Comment:
Note that `brew` doesn't add the runtime dependencies of build dependencies to `PATH`, etc, so this might make it harder to find `python3`.

Reply:
At least `llnode` doesn't directly use Python command in build scripts. It is just that `node` runs Python and I assume our `node` test should be similar to behavior we want.